--- origsrc/mingw-w64-v3.0.0/mingw-w64-tools/widl/Makefile.am	2012-11-14 03:19:32.000000000 -0600
+++ src/mingw-w64-v3.0.0/mingw-w64-tools/widl/Makefile.am	2014-01-01 19:08:36.080512900 -0600
@@ -56,7 +56,7 @@ widl_SOURCES = src/widl.h \
   include/winnls.h \
   include/winnt.rh
 
-widl_CPPFLAGS = -I$(top_srcdir)/include -DDEFAULT_INCLUDE_DIR=\"$(includedir)/../$(target)/include\"
+widl_CPPFLAGS = -I$(top_srcdir)/include -DDEFAULT_INCLUDE_DIR=\"$(prefix)/$(target)/sys-root/mingw/include\"
 widl_CFLAGS = -O3 -g -Wall -Wformat -Wpacked -Wmissing-declarations -Wimplicit-function-declaration -Wmissing-prototypes -Wstrict-aliasing=2
 
 DISTCHECK_CONFIGURE_FLAGS = --host=$(host) --target=$(target)
